### ISSUE TITLE
Jar

### ISSUE THREAD

**Issue by philiprbrenan** [vlambo3/TaskManagerApp#1]
Might it be possible to create a java jar file so that a user who does not have an ide can just download the jar file and run it with ```java tm.jar``` and then interact with the running program via their web browser?  It might also be useful to have a github action that demonstrates this code and builds the jar file as an artifact of the  github action.   This would have the benefit of making your excellent work available to a wider audience.

**Comment by vlambo3** [comment]
Considering the potential size constraints of JAR files, I've opted not to include one directly in the repository. However, to ensure smooth distribution and collaboration, I've configured a GitHub Action workflow. This workflow automates the compilation process, creating a downloadable JAR artifact. This approach optimizes repository performance, grants collaborators easy access to the latest JAR version, and streamlines deployment. Additionally, it leverages GitHub's version control to maintain consistency and guarantee the use of the correct JAR for each code version. This scalable approach adapts effectively as the project evolves and the JAR file size increases.
